src/test/components/ArticleCard.test.ts

```
@@ -0,0 +1,150 @@
+    expect(truncatedDescription).toBe(
+      longDescription.substring(0, 120) + "...",
+    );
+  });
+
+  it("devrait garder la description courte intacte", () => {
+    const shortDescription = "Description courte";
+
+    const truncatedDescription =
+      shortDescription.length > 120
+        ? shortDescription.substring(0, 120) + "..."
+        : shortDescription;
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)